File: Java/프로그래머스/3/42884. 단속카메라/단속카메라.java
import java.util.*;
class Solution {
    public int solution(int[][] routes) {
        Arrays.sort(routes, (a, b) -> a[1] - b[1]);
        int answer = 1;
        int point = routes[0][1];
        for (int[] route : routes) {
            if (route[0] > point) {
                answer++;
                point = route[1];
            }
        }
        return answer;
    }
}

File: Python/프로그래머스/3/42884. 단속카메라/단속카메라.py
def solution(routes):
    answer = 1
    routes = sorted(routes, key=lambda x: x[1])
    print(routes)
    min_num = routes[0][1]
    for a, b in routes:
        if min_num < a:
            answer += 1
            min_num = b
    return answer